dqn.py

```diff
@@ -202,28 +234,40 @@ def main(config: RLConfig) -> None:
     batch_size = config.batch_size
     global_steps = 0
     average_steps = []
+    rewards = []
+    episode_reward = 0
 
     for episode in range(config.num_episodes):
         state = env.reset()
 
         steps = 0
         while True:
-            agent.set_epsilon(linear_epsilon_anneal(global_steps))
+            agent.set_epsilon(
+                linear_epsilon_anneal(global_steps, config.exploration_steps))
+
             action = agent.sample_action(state)
             next_state, reward, done, _ = env.step(action)
             agent.memorize(state, action, reward, next_state, done)
+            episode_reward += reward * (config.discount_factor**steps)
             state = next_state
             steps += 1
 
             if done:
                 average_steps.append(steps)
-                if episode % 20 == 0:
+                rewards.append(episode_reward)
+
+                if (episode + 1) % 20 == 0:
                 # if episode % 1 == 0:
-                    log.info(f"Episode: {episode:05d}/{config.num_episodes:05d}\t\t "
-                             f"Score: {np.mean(average_steps):.1f}\t\t "
-                             f"Max-|Q|: {agent.get_max_q_val():.1f}\t\t "
+                    log.info(f"Episode: {episode + 1:5d}/{config.num_episodes:5d}\t\t "
+                             f"#Steps: {np.mean(average_steps):7.1f}\t\t "
+                             f"Reward: {np.sum(rewards):7.1f}\t\t "
```

Comment:
Why sum all the episode rewards rather than mean?

Reply:
Leftover from previous code, when I didn't calculate a per-episode total reward, but accumulated all rewards in a single list.
I'll correct that.